vuepress/docs/next/tutorials/getting-started/gke-install.md

```diff
@@ -2,174 +2,149 @@
-kube-system       Active   6m13s
 
-```
+2. Go to `Kubernetes Engine` → `Clusters`
 
-### Install the NGINX Ingress Controller
+3. Select an existing project or create a new one
 
-Entando isn’t compatible out of the box  with the default ingress controller provided in GKE.
-See here for more if you’re interested in GKE ingress: <https://cloud.google.com/kubernetes-engine/docs/concepts/ingress>
+4. Click `Enable` for the `Kubernetes Engine API`
 
-We’re going to install the NGINX ingress controller to manage the ingresses for Entando services
-deployed by the operator. This will be a simpler and more adaptable configuration for most users and
-environments. Users who really need the GKE ingress controller (rare) could integrate it following
-the instructions provided by GKE and then customize the service definition created by the Entando
-operator.
+5. Once the API is enabled, click `Create` to create a cluster
 
-These are the minimal instructions to prepare NGINX ingress using the Google Cloud Shell. To install it
-using your local `kubectl` or to vary other settings please see the more detailed documents here:
-<https://cloud.google.com/community/tutorials/nginx-ingress-gke> and <https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke>.
+6. Click the `Configure` button for the `GKE Standard` option. Unless otherwise indicated, you can keep the default configuration options.
 
-1. Initialize your user as a cluster-admin:
-```
+7. In the left menu, select `default-pool` → `Nodes`
+
+8. Select "e2-standard-2" as the Machine Type if you are setting up a basic test cluster for a single Entando Application. Additional CPU and memory may be required for a shared cluster containing multiple Entando Applications or to improve performance. Refer to [Appendix A](#appendix-a-configuring-clustered-storage) for details on clustered storage.
+
+9. Click `Create`. It may take a few minutes for the cluster to initialize. 
+
+10. Click `Connect`
+
+11. Click `Run in Cloud Shell`. Alternatively, connect your local `kubectl` to the GKE cluster.
+
+12. Run `kubectl get node` to verify your connection. The output should list the nodes in your cluster.
+
+### Install the NGINX Ingress Controller
+
+The following steps install the NGINX ingress controller to manage the ingresses for Entando services deployed by the operator. This is a simpler and more adaptable configuration for most users and environments. Users who require the GKE ingress controller (this is rare) can follow
+[the integration instructions provided by GKE](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress) and then customize the service definition created by the Entando operator.
+
+These are the mimimum instructions to prepare the NGINX ingress using the Google Cloud Shell. Refer to <https://cloud.google.com/community/tutorials/nginx-ingress-gke> and <https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke> for installation using your local `kubectl` or to vary other settings.
```

Comment:
second sentence, change links to text? easier to absorb >> For installation using your local `kubectl` or to vary other settings, refer to [Ingress with NGINX](https://cloud.google.com/community/tutorials/nginx-ingress-gke) and [GCE-GKE details](https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke)